### DISCUSSION TITLE
Differences in Light Chain Detection: MiXCR 3.0.13 vs. MiXCR 4.6.0

### DISCUSSION THREAD

**Discussion by gsr9999** [liulab-dfci/TRUST4#334] [Q&A]
Hello MiXCR Support Team,

We are currently using MiXCR version 3.0.13 to analyze BCR sequencing data generated with Takara’s SMART-Seq v4 chemistry. To modernize our pipeline, we are evaluating MiXCR version 4.6.0 and have conducted a comparative analysis by running both versions on the same set of FASTQ files derived from 36 plates of SMART-Seq data (human samples).
In most wells across the plates (96% or 1904 wells), both versions successfully made heavy or light chain calls. However, in 45 wells, MiXCR 3.0.13 detected productive light chains, whereas MiXCR 4.6.0 did not yield any corresponding sequences.
To investigate, I reviewed the sequences from these 45 wells using IMGT/V-QUEST, and all appear to be valid. The IMGT annotations indicate:
•       V-DOMAIN Functionality: Productive
•       CDR and FR lengths: Within expected ranges
•       JUNCTION Frame: In-frame

This raises the question: why are these 45 light chains not being identified by MiXCR 4.6.0?
<img width="210" alt="venn" src="https://github.com/user-attachments/assets/0fe9a40f-8b7c-45dd-9195-b5fde040975f">

Example Case:
One of the affected wells, Plate99_Well_A01, was processed with both versions.
•       MiXCR 3.0.13 Results:
o       targetSequences column:
GGGGGGGTCACAAGAGGCAGCGCTCTCGGGACGTCTCCACCATGGCCTGGGCTCTGCTGCTCCTCACCCTCCTCACTCAGGACACAGGGTCCTGGGCCCAGTCTGCCCTGACTCAGCCTGCCTCCGTGTCTGGGTCTCCTGGACAGTCGATCACCATCTCCTGCACTGGAACCAGCAGTGATGTTGGGAGTTATAACCTTGTCTCCTGGTACCAACAGCACCCAGGCAAAGCCCCCAAACTCATAATTTATGAGGTCAGTCAGCGGCCCTCAGGGGTTTCTTATCGCTTCTCTGGCTCCAAGTCTGGCAACACGGCCTCCCTGACAATCTCGAGGCTCCAGGCTGAGGACGAGGCTGATTATTACTGCTGCTCATTTGCAGGTAGTAGCACTTTCCATTATGTCTTCGGAACTGGGACCACGGTCACCGTCCTAGGTCAGCCCAAGGCCAACCCCACTGTCACTCTGTTCCCGCCCTCCTCTGAGGAGCTTCAAGCCAAC
o       Clone Count: 600
o       IMGT Annotation: IGLV2-23*02
•       MiXCR 4.6.0 Results:
o       No output for this well.

Commands Used:
•       MiXCR 3.0.13:
mixcr align -s hsa -p kAligner2 240909_P99_A01_S35_R1.fastq.gz 240909_P99_A01_S35_R2.fastq.gz "${output_vdjca_file}"
mixcr assemble --write-alignments "${output_vdjca_file}" "${output_clna_file}"
mixcr assembleContigs "${output_clna_file}" "${output_clns_file}"
mixcr exportClones -n 3 -p fullImputed "${output_clns_file}" "${output_clones_TXT_file}"
•       MiXCR 4.6.0:
mixcr analyze smart-seq2-vdj --species hsa -Massemble.consensusAssemblerParameters.assembler.maxConsensuses=3 240909_P99_A01_{{CELL0ROW:a}}{{CELL0COL:a}}_{{R}}.fastq.gz "240909_P99_A01"
Additional Information:
Attached is a Venn diagram summarizing the overlap and differences between the outputs of MiXCR 3.0.13 and MiXCR 4.6.0.

Could you help clarify why MiXCR 4.6.0 fails to make these calls despite the sequences appearing valid? Are there changes in the algorithms or parameters that might explain this discrepancy?

**Comment by mourisl** [comment]
I think you may ask this in the MIXCR repository, e.g.: https://github.com/milaboratory/mixcr/

**Reply by gsr9999** [reply]
ok, thank you
